net-misc/ytdl-gui/ytdl_gui-2.0.recipe

```diff
@@ -0,0 +1,80 @@
+SUMMARY="A simple-to-use, cross-platform graphical interface for youtube-dl"
+DESCRIPTION="YouTubeDL-GUI is a cross-platform graphical interface for \
+youtube-dl, a downloader for online media sharing services like YouTube,\
+ Vimeo, SoundCloud, Bandcamp, and more. It is based on Qt and aims to be\
+ simple-to-use, while still providing various options like file formats \
+and quality levels."
+HOMEPAGE="https://github.com/JaGoLi/ytdl-gui"
+COPYRIGHT="2020 JaGoLi"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitTag="2.0"
+SOURCE_URI="https://github.com/JaGoLi/ytdl-gui/archive/$srcGitTag.tar.gz"
```

Comment:
Didn't check it, but maybe you could skip $srcGitTag and only use $portVersion in SOURCE_URI?

Reply:
Done in a new PR instead:
https://github.com/haikuports/haikuports/pull/5670